lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3400,24 +3383,26 @@ balanceTx
     -> TimeTranslation
     -> StdGenSeed
     -> PartialTx era
-    -> Either
-        ErrBalanceTx (Cardano.Tx era)
+    -> Either ErrBalanceTx (Cardano.Tx era)
 balanceTx
-    (Wallet' utxoAssumptions utxo (AnyChangeAddressGenWithState genChange s))
-    pp
-    timeTranslation
-    seed
-    ptx =
-    fmap fst $ flip evalRand (stdGenFromSeed seed) $ runExceptT $
-        balanceTransaction @_ @(Rand StdGen)
-            (nullTracer @(Rand StdGen))
-            utxoAssumptions
-            pp
-            timeTranslation
-            (constructUTxOIndex utxo)
-            genChange
-            s
-            ptx
+    ( Wallet'
+        utxoAssumptions
+        utxo
+        (AnyChangeAddressGenWithState genChange s)
+    )
+    pp timeTranslation seed partialTx =
```

Comment:
This re-wrapping seems to be going in the opposite direction to the changes to `Write.Tx.Balance.balanceTransaction`. My suggestion would be to revert to one `\n` per argument. Particularly when the first argument needs multiple lines anyway.